rclcpp/include/rclcpp/subscription_base.hpp

```diff
@@ -138,6 +138,12 @@ class SubscriptionBase : public std::enable_shared_from_this<SubscriptionBase>
   void
   handle_message(std::shared_ptr<void> & message, const rmw_message_info_t & message_info) = 0;
 
+  // TODO(karsten1987): Does it make sense to pass in a weak_ptr?
```

Comment:
A `weak_ptr` to what? Can you please create an issue to follow up on this...

Reply:
I was wondering if a `weak_ptr` could help around placing a custom deleter in the shared pointer for the `any_callback_.dispatch()` function. But the solution implemented here does work, so I don't think we need that TODO any longer. 
I'll address that with the other documentation touchups in a follow up PR